apps/docs/app/root/layout/SiteHeader.tsx

```diff
@@ -71,12 +77,8 @@ export const SiteHeader = () => {
       <MobileNavigation
```

Comment:
Vi savner fortsatt en header element i semantikken av side så i linje 68 hadde jeg brukt  as="header" for å fikse det